src/Dscp/Resource/Network.hs

```diff
@@ -0,0 +1,94 @@
+-- | ZMQ TCP Networking resource allocation.
+
+module Dscp.Resource.Network where
+
+import Universum
+
+import Control.Lens (makeLenses)
+import Control.Monad.Component (buildComponent)
+import Loot.Base.HasLens (HasLens (..))
+import Loot.Log (Level)
+import Loot.Network.ZMQ (ZTGlobalEnv, ZTNetCliEnv, ZTNetServEnv, ZTNodeId (..), createNetCliEnv,
+                         createNetServEnv, ztGlobalEnv, ztGlobalEnvRelease)
+
+import Dscp.Resource.Class (AllocResource (..))
+
+----------------------------------------------------------------------------
+-- Common
+----------------------------------------------------------------------------
+
+-- TODO DSCP-105 Use log-warper here. Perhaps i should just put
+-- logging function into parameters?
+tempNetLog :: Level -> Text -> IO ()
+tempNetLog l t = putTextLn $ "[" <> show l <> "]: " <> t
+
+-- TODO Should I also release sockets?
+
+----------------------------------------------------------------------------
+-- Client
+----------------------------------------------------------------------------
+
+-- | Client networking params.
+data NetCliParams = NetCliParams
+    { ncPeers :: Set ZTNodeId
```

Comment:
Why don't you make your datatypes strict? (Not only here, but in many other places)

Reply:
@volhovm cho po fixam